doc/rados/operations/placement-groups.rst

```diff
@@ -62,154 +69,167 @@ To ``get`` the value of the flag use the command:
-have a ratio of 3.0, while a k=4,m=2 erasure coded pool will have a
-ratio of 1.5.
-
-**RAW CAPACITY** is the total amount of raw storage capacity on the
-OSDs that are responsible for storing this pool's (and perhaps other
-pools') data.  **RATIO** is the ratio of that total capacity that
-this pool is consuming (i.e., ratio = size * rate / raw capacity).
-
-**TARGET RATIO**, if present, is the ratio of storage that the
-administrator has specified that they expect this pool to consume
-relative to other pools with target ratios set.
-If both target size bytes and ratio are specified, the
-ratio takes precedence.
-
-**EFFECTIVE RATIO** is the target ratio after adjusting in two ways:
-
-1. Subtracting any capacity expected to be used by pools with target size set
-2. Normalizing the target ratios among pools with target ratio set so
-   they collectively target the rest of the space. For example, 4
-   pools with target_ratio 1.0 would have an effective ratio of 0.25.
-
-The system uses the larger of the actual ratio and the effective ratio
-for its calculation.
-
-**BIAS** is used as a multiplier to manually adjust a pool's PG based
-on prior information about how much PGs a specific pool is expected
-to have.
-
-**PG_NUM** is the current number of PGs for the pool (or the current
-number of PGs that the pool is working towards, if a ``pg_num``
-change is in progress).  **NEW PG_NUM**, if present, is what the
-system believes the pool's ``pg_num`` should be changed to.  It is
-always a power of 2, and will only be present if the "ideal" value
-varies from the current value by more than a factor of 3 by default.
-This factor can be be adjusted with:
+- **POOL** is the name of the pool. 
 
-.. prompt:: bash #
+- **SIZE** is the amount of data stored in the pool. 
+  
+- **TARGET SIZE** (if present) is the amount of data that is expected to be
+  stored in the pool, as specified by the administrator. The system uses the
+  greater of the two values for its calculation.
+
+- **RATE** is the multiplier for the pool that determines how much raw storage
+  capacity is consumed. For example, a three-replica pool will have a ratio of
+  3.0, and a ``k=4 m=2`` erasure-coded pool will have a ratio of 1.5.
+
+- **RAW CAPACITY** is the total amount of raw storage capacity on the specific
+  OSDs that are responsible for storing the data of the pool (and perhaps the
+  data of other pools). 
+
+- **RATIO** is the ratio of (1) the storage consumed by the pool to (2) the
+  total raw storage capacity. In order words, RATIO is defined as (SIZE * RATE)
+  / RAW CAPACITY.
+
+- **TARGET RATIO** (if present) is the ratio of the expected storage of this
+  pool (that is, the amount of storage that this pool is expected to consume,
+  as specified by the administrator) to the expected storage of all other pools
+  that have target ratios set.  If both ``target_size_bytes`` and
+  ``target_size_ratio`` are specified, then ``target_size_ratio`` takes
+  precedence.
```

Comment:
@anthonyeleven - There were a few sentences we were a little uncertain about. This ("If both...") was one. It originally read: "If both target size bytes and ratio are specified, the ratio takes precedence." Does this revision look okay to you? Thanks for your help.

Reply:
Honestly the autoscaler is a mystery to me so I have no idea :-/